SPEC.md

```diff
@@ -576,13 +577,14 @@ multi-line """[\n]
 
 #### Interaction with Whitespace Escapes
 
-Multi-line strings support the same mechanism for escaping whitespace
-as Quoted Strings.
-When processing a Multi-line String, implementations MUST dedent the string _after_
-resolving all whitespace escapes, but _before_ resolving other backslash escapes.
-Furthermore, a whitespace escape that attempts to escape the final line's newline
-and/or whitespace prefix is invalid since the multi-line string has to still be
-valid with the escaped whitespace removed.
+Multi-line strings support the same mechanism for escaping whitespace as Quoted
+Strings.
+
+When processing a Multi-line String, implementations MUST dedent the string
+_after_ resolving all whitespace escapes, but _before_ resolving other backslash
+escapes. Furthermore, a whitespace escape that attempts to escape the final
+line's newline and/or whitespace prefix is invalid since the multi-line string
```

Comment:
I'm confused by this phrasing. It seems to say that escaped newline and/or whitespace are not allowed as the multi-line string has to be valid after the escaped whitespace is removed, but the text below and two of the new tests actually allow escaped whitespace in the final line.  
This seems to imply that an escaped newline and/or whitespace are only not allowed if the multiline string would become invalid, but that's not how I read this sentence.

Note that would also allow escaped newlines in some cases, e.g.

```kdl
node """
  lorem
  ipsum
  \
    """
```

which would be equivalent to

```kdl
node """
  lorem
  ipsum
  """
```

which would imo be worth adding as test.

Reply:
@bgotink [this test sort of tests that](https://github.com/kdl-org/kdl/blob/main/tests/test_cases/input/multiline_string_whitespace_only.kdl#L5) but having a more explicit one would be good.

I believe the bit that says that a trailing `\` is invalid actually _intended_ to refer to a case like this:

```kdl
"""
lorem
ipsum\
"""
```

which would be invalid because:

```kdl
"""
lorem
ipsum"""
```

This should probably be reworded, assuming these are the semantics we want to keep.

I'm not terribly inclined to change multiline strings any further, though, tbh.

Reply:
> I'm not terribly inclined to change multiline strings any further,

Oh no, I'm definitely not asking to change multiline strings! I was just confused about the text vs tests.

Reply:
oh no that's fine. I was also mostly responding to the ongoing convo with @eilvelia who brought up a lot of good points, though I'm leading towards "clarify, don't change"